components/SideBar.tsx

```diff
@@ -60,33 +67,35 @@ const Sidebar = () => {
         >
           <ExitIcon />
         </button>
-        <div className="text-xs	flex flex-col mt-6 justify-between flex-1">
+        <div className="text-xs	flex flex-col mt-6 justify-between flex-1 text-ocf-yellow">
           <MenuLink
             linkProps={{ href: '/dashboard' }}
             label="Dashboard"
             svg={<DashboardIcon />}
-            textColor="text-amber"
+            textColor={getTextColor('/dashboard')}
```

Comment:
The `getTextColor` function is really nice, but I wonder if we can put the getTextColor function call inside `<MenuLink` to reduce repetition. The href is accessible inside that component.